pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/controller/PinotControllerTransportFactory.java

```diff
@@ -56,4 +71,18 @@ public SSLContext getSslContext() {
   public void setSslContext(SSLContext sslContext) {
     _sslContext = sslContext;
   }
+
+  public PinotControllerTransportFactory withConnectionProperties(Properties properties) {
+    _readTimeout = Integer.parseInt(properties.getProperty("controllerReadTimeout",
```

Comment:
Same as before, property names should reflect time units if it is not configurable